src/Domain/Value/PhpVersion.php

```diff
@@ -40,11 +30,6 @@ public static function fromString(string $version): self
         return new self($version);
     }
 
-    public function isStable(): bool
```

Comment:
isn't this used? 🤔

Reply:
No

Reply:
Ok I thought we used it in the template, but maybe we can keep it when 8.1 or nightly will be tested

Reply:
That way I would see if we could do this in another way.
Maybe isStable = not nightly, or isStable = not 8.1, or something else...

By the way is was written, we forgot to update the code to add 8.0 in the list of stable php.
And there was 7.0, 7.1, 7.2 which we don't support anymore.